Screenshot Maker.cpp

```diff
@@ -56,6 +67,7 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
 				"  -e, -encoder png          file encoder: bmp/jpeg/gif/tiff/png (default: png)\n"
 				"  -q, -quality 100          file quality for jpeg (between 0 and 100)\n"
 				"  -r, -resize 50            image size, % of the original size (between 1 and 99)\n\n"
```

Comment:
```suggestion
				"  -r, -resize 50            image size, % of the original size (between 1 and 99)\n"
```

Reply:
Good catch. It wasn't possible for me to test this. There's no console output, because the project targets the Windows Subsystem, so there's no console that this output goes to.

I will need AttachConsole() to output error messages. https://learn.microsoft.com/en-us/windows/console/attachconsole